build/ps-entrypoint.sh

```diff
@@ -309,6 +315,7 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 
 			CREATE USER 'replication'@'%' IDENTIFIED BY '${REPLICATION_PASSWORD}';
 			GRANT SYSTEM_USER, REPLICATION SLAVE ON *.* to 'replication'@'%';
+			GRANT SELECT ON performance_schema.threads to 'replication'@'%';
```

Comment:
Can we load it only in case if GR is enabled ?

Reply:
And for backups we need to add GROUP_REPLICATION_ADMIN for xtrabackup user.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/91/commits/037bb02d3f7b8602ac82cf519aa75d2217a61b20